resources/application-connector-helper/OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
```

Comment:
I would move piotrmsc, kubadz, strekm, jakkab, Tomasz-Smelcerz-SAP and Demonsthere to reviewers section